views/pages/db.ejs

```diff
@@ -16,6 +16,10 @@
       <ul>
           <% results.forEach(function(r) { %>
               <li><%= r.id %> - <%= r.name %></li>
+              <td><%= r.prod_name%></td>
+              <td><%= r.prod_type%></td>
+              <td><%= r.prod_expiry%></td>
+              <td><%= r.prod_quantity%></td>
```

Comment:
The changes in this file aren't actually necessary to commit as it's just a testing file, and also the "prod_xxx" parameters won't actually exist for this results object since it refers to the test_table table which just has "id" and "name" as keys. Could you delete these four lines between 19 and 22, and then recommit this file to reverse the changes?

Reply:
forgot to remove these, thank you!